README.md

```diff
@@ -72,21 +72,21 @@ This tool is only compatible with its own export files.
 ## Use
 
 ### Export
-`sudo -u www-data ./bin/sugarcrm systemdata:export:teams /tmp`<br />
-`sudo -u www-data ./bin/sugarcrm systemdata:export:roles /tmp`<br />
-`sudo -u www-data ./bin/sugarcrm systemdata:export:users /tmp`<br />
-`sudo -u www-data ./bin/sugarcrm systemdata:export:reports /tmp`<br />
-`sudo -u www-data ./bin/sugarcrm systemdata:export:awf /tmp`<br />
+`sudo -u www-data php ./bin/sugarcrm systemdata:export:teams /tmp`<br />
```

Comment:
@xye do we need this? Is it best practice? It does work also without it as far as I know, correct?